.husky/pre-push.js

```diff
@@ -0,0 +1,14 @@
+const inquirer = require('inquirer');
```

Comment:
Should we add `inquirer` specifically in our devDependencies instead of relying on the transient dependencies used in Lerna and electron-windows-store?

Seems like as of major version 9, this module is ESM-only as well which will break this code if our dependency tree ends up relying on it eventually.

Reply:
Yes, please add an explicit devDependency if there's not already a direct dependency in some other subpackage.